manifests/profile/cache_rpms.pp

```diff
@@ -15,13 +19,23 @@
     timeout   => '600',
     logoutput => false,
     require   => Yumrepo['epel'],
```

Comment:
It's because the epel yumrepo was happening after the packages were cached and epel was the only one we were adding. I'm not sure how we'd avoid that in this case. I think `include epel` is used in multiple places.